Lib/test/test_asyncio/test_tasks.py

```diff
@@ -4,6 +4,7 @@
 import contextlib
 import functools
 import io
+import gc
```

Comment:
you're too kind :)  Will keep that in mind if we run into any other cpython issues.